backend/handlers/handlers.go

```diff
@@ -203,6 +203,7 @@ func NewHandlers(
 	getAPIRouter(apiRouter)("/coins/ltc/headers/status", handlers.getHeadersStatus(coinpkg.CodeLTC)).Methods("GET")
 	getAPIRouter(apiRouter)("/coins/btc/headers/status", handlers.getHeadersStatus(coinpkg.CodeBTC)).Methods("GET")
 	getAPIRouter(apiRouter)("/coins/btc/set-unit", handlers.postBtcFormatUnit).Methods("POST")
+	getAPIRouter(apiRouter)("/coins/btc/parseExternalAmount", handlers.getBTCParseExternalAmount).Methods("GET")
```

Comment:
why not put this into the `backend/coins/btc/handlers/` package since it's about the btc coin and not a general backend method?

Reply:
Put here since there was already `/coins/btc/set-unit` which seems related to the same scope. Could also move them both though..

Reply:
I actually wrote this comment and wanted to delete it. this is the right place for this after all, as it is not account-specific!